samples/interactive-tutorials/prediction/get_prediction.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START prediction_for_products]
```

Comment:
Please use the correct product prefix `retail` in region tags
```suggestion
# [START retail_prediction_get_prediction]
```

Reply:
Fixed